whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -205,10 +227,44 @@ private Boolean isRedirect(ObjectType objectType, final String key) {
     }
 
     protected Response lookupObject(final HttpServletRequest request, final Set<ObjectType> objectTypes, final String key) {
-        List<RpslObject> result =  rdapQueryHandler.handleQuery(getQueryObject(objectTypes, key), request);
+        Iterable<RpslObject> result =  rdapQueryHandler.handleQuery(getQueryObject(objectTypes, key), request);
         return getResponse(request, result);
     }
 
+    protected Response lookupForOrganisation(final HttpServletRequest request, final Set<ObjectType> objectTypes,
+                                             final String key) {
+        final List<RpslObject> organisationStream = rdapQueryHandler.handleQuery(getQueryObject(objectTypes,
```

Comment:
organisationStream isn't actually a stream, it's a list. better to use the method that returns a stream.